.github/workflows/docs.yaml

```diff
@@ -0,0 +1,14 @@
+name: Documentation Updates
+
+on: [push]
```

Comment:
I believe it will, the `pull_request` event in my understanding is specially about PRs as such, you can write a bot that comments on certain PRs, for example. Push is a more general one. We can test this once it's merged to be sure :)